.github/workflows/sdk-python-ci.yml

```diff
@@ -1,25 +1,21 @@
 name: CI Pipeline for SDK - Python
 
-on:
-  push:
-    branches: ["main"]
-
+on: [push, pull_request]
 
 jobs:
   test:
-#    needs: ["gen"]
-    uses: openclimatefix/.github/.github/workflows/python-test.yml@working-directory
+    uses: openclimatefix/.github/.github/workflows/python-test.yml@python-options
```

Comment:
if it works, lets merge it in, in .github repo

Reply:
https://github.com/openclimatefix/.github/pull/32

Reply:
https://github.com/openclimatefix/.github/pull/33

Reply:
See 7a42930b28c4e11c076a097fbda45b8fc6f8e134